src/components/LegendWrapper/test.cypress.tsx

```diff
@@ -0,0 +1,101 @@
+/// <reference types="cypress" />
+
+import React from 'react';
+import { mount } from '@cypress/react';
+
+import TestWrapper, { TestThemeWrapper } from '../TestWrapper';
+import { Box, Checkbox, Heading, LegendWrapper, Radio, crukTheme, su2cTheme } from '../';
+
+const content = () => {
+  return (
+    <>
+      <section>
+        <Heading h3 textSize="l">
+          Legend Wrapper
+        </Heading>
+        <Box marginBottom="m">
+          <LegendWrapper legend="Legend Example" />
+        </Box>
+      </section>
+      <section>
+        <Heading h3 textSize="l">
+          Required
+        </Heading>
+        <Box marginBottom="m">
+          <LegendWrapper legend="Legend Example" required />
+        </Box>
+      </section>
+      <section>
+        <Heading h3 textSize="l">
+          Radio
+        </Heading>
+        <Box marginBottom="m">
+          <LegendWrapper legend="Legend Example">
+            <Radio name="example1" value="one">
+              Option one
+            </Radio>
+          </LegendWrapper>
+        </Box>
+      </section>
+
+      <section>
+        <Heading h3 textSize="l">
+          Radio Error
+        </Heading>
+        <Box marginBottom="m">
+          <LegendWrapper legend="Legend Example" hasError errorMessage="Error Message">
+            <Radio name="example1" value="one">
+              Option one
+            </Radio>
+          </LegendWrapper>
+        </Box>
+      </section>
+      <section>
+        <Heading h3 textSize="l">
+          Checkbox
+        </Heading>
+        <Box marginBottom="m">
+          <LegendWrapper legend="Legend Example">
+            <Checkbox disabled={false} value="value" />
+          </LegendWrapper>
+        </Box>
+      </section>
+
+      <section>
+        <Heading h3 textSize="l">
+          Checkbox Error
+        </Heading>
+        <Box marginBottom="m">
+          <LegendWrapper legend="Legend Example" hasError errorMessage="Error Message">
+            <Checkbox disabled={false} value="value" />
+          </LegendWrapper>
+        </Box>
+      </section>
+    </>
+  );
+};
+
+describe('TextField', () => {

```

Comment:
change to Legend wrapper

Reply:
delete cyp[ress snap with this label cypress/snapshots/LegendWrapper/test.cypress.tsx/TextField -- should match snapshot.snap.png